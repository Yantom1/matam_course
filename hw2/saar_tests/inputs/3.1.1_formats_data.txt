32305629  , Super Woman, 8, 2012, 1
33401550, Dora the Explorer, 19, 2000, 2
11401830,  Homer          Simpson  , 50 ,1970, 3
12345678,   Leonardo    da    Vinci   ,  25 , 1995 , 3
27804448  ,             Aviv     Censo    r       , 20, 2000, 15
