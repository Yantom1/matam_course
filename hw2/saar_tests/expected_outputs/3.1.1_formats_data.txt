11401830, Homer Simpson, 50, 1970, 3
12345678, Leonardo da Vinci, 25, 1995, 3
27804448, Aviv Censo r, 20, 2000, 15
